apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -170,7 +170,7 @@
         margin-bottom: f.spacing(i);
     }
 }
-.c-contentPage-content a {
+.c-content-container > p > a {
```

Comment:
are all generated links on the pages nested within p tags?

Reply:
I think I'd be really careful with this. what was the reason for the change?

Reply:
`c-contentPage-content a`overwrites the text in the cards so that it's a blue link.
Even if I put a class directly on the link text and set the text to black it stays blue

<img width="867" alt="image" src="https://user-images.githubusercontent.com/24740015/222775260-3e29cf9b-7d1c-44fd-8339-badf8da40b1a.png">